src/main/java/com/tabnine/userSettings/AppSettingsComponent.kt

```diff
@@ -19,6 +19,7 @@ class AppSettingsComponent {
     private val suggestionsModeService = DependencyContainer.instanceOfSuggestionsModeService()
     private val logFilePathComponent = JBTextField()
     private val logLevelComponent = JBTextField()
+    private val debounceTimeComponent = JBTextField()
```

Comment:
are u sure you want to expose this? i think we need to come up with the correct value ourselves, not sure it should be in the user settings panel.
if you want to easily configure it, put it in a secret file or env var, no?

Reply:
I want to let us play with this value easily to get the correct default value.
also, I'm not sure this is a bad idea to add it, since some users type faster and some slower, and they can adapt this value to their needs.